DEVELOPMENT.md

```diff
@@ -42,133 +48,84 @@ At this point, you may find it useful to return to these `Tekton Pipeline` docs:
+### Install tools
 
-- Create a secret
-```yaml
-kubectl create secret docker-registry ${SECRET_NAME} \
-  --docker-username=${USERNAME} \
-  --docker-password=${PASSWORD} \
-  --docker-email=me@here.com \
-  --namespace=tekton-pipelines
-```
+You must install these tools:
 
-- Add it to serviceAccount
-
-Because you will install tekton-pipelines use `ko` later, before that, you need to change [the contents of serviceAccount](./config/200-serviceaccount.yaml) to below:
-```yaml
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: tekton-pipelines-controller
-  namespace: tekton-pipelines
-  labels:
-    app.kubernetes.io/component: controller
-    app.kubernetes.io/instance: default
-    app.kubernetes.io/part-of: tekton-pipelines
-imagePullSecrets:
-  - name: ${SECRET_NAME}
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: tekton-pipelines-webhook
-  namespace: tekton-pipelines
-  labels:
-    app.kubernetes.io/component: webhook
-    app.kubernetes.io/instance: default
-    app.kubernetes.io/part-of: tekton-pipelines
-imagePullSecrets:
-  - name: ${SECRET_NAME}
-```
+1.  [`git`](https://help.github.com/articles/set-up-git/): For source control
 
-After setting `GOPATH` and putting `$GOPATH/bin` on your `PATH`, you must then install these tools:
+1.  [`go`](https://golang.org/doc/install): The language Tekton Pipelines is
+    built in.
+    > **Note** Golang [version v1.15](https://golang.org/dl/) or higher is recommended.
 
-3.  [`ko`](https://github.com/google/ko): For development. `ko` version v0.5.1 or
+1. [`ko`](https://github.com/google/ko#install): The Tekton project  uses `ko` to simplify the building of its container images from `go` source, push these images to the configured image repository and deploy these images into Kubernetes clusters.
+    > **Note** `ko` version v0.5.1 or
     higher is required for `pipeline` to work correctly.
-    
-4.  [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
-    interacting with your kube cluster
-
-The user you are using to interact with your k8s cluster must be a cluster admin
-to create role bindings:
-
-```shell
-# Using gcloud to get your current user
-USER=$(gcloud config get-value core/account)
-# Make that user a cluster admin
-kubectl create clusterrolebinding cluster-admin-binding \
-  --clusterrole=cluster-admin \
-  --user="${USER}"
-```
-
-### Install in custom namespace
 
-1.  To install into a different namespace you can use this script :
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
+    interacting with your Kubernetes cluster
 
-```shell
-#!/usr/bin/env bash
-set -e
+    > :warning: The user interacting with your K8s cluster **must be a cluster admin** to create role bindings.
 
-# Set your target namespace here
-TARGET_NAMESPACE=new-target-namespace
+    **Setting cluster admin role example**:
```

Comment:
is this something we can drop from here? or at least delay it until machine specific recommendations 🤔

Reply:
I reviewed the changes again and this does not apply here unless I am missing something. Not blocking this PR though, this can be revisited in later iteration.